.github/workflows/post_to_mastodon.sh

```
@@ -0,0 +1,26 @@
+#!bin/bash
+
+version="${${{ github.event.pull_request.title }}##* }"
```

Shellcheck (0.10.0)

[error] To expand via indirection, use arrays, ${!name} or (for sh only) eval.

(SC2082)